docs/architecture/adr/README.md

```diff
@@ -38,3 +38,4 @@ We document our decisions using [Architecture Decision Records](https://github.c
 - [ADR #033 - NLB for mTLS](ADR033-nlb-for-mtls.md)
 - [ADR #034 - Single service operator](ADR034-one-service-operator-different-resource-kinds.md)
 - [ADR #035 - Aurora postgres](ADR035-aurora-postgres.md)
+- [ADR #037 - Per namespace istio gateways](ADR037-per-namespace-gateways.md)
```

Comment:
What happened to ADR 036?

Reply:
Sam and Phil have that reserved in another branch.

Reply:
Reserved for #411